modules/ximgproc/samples/thinning.cpp

```diff
@@ -19,8 +19,8 @@ int main()
 
     /// Apply thinning to get a skeleton
     Mat img_thinning_ZS, img_thinning_GH;
-    //ximgproc::thinning(img_binary, img_thinning_ZS, THINNING_ZHANGSUEN);
-    //ximgproc::thinning(img_binary, img_thinning_GH, THINNING_GUOHALL);
+    ximgproc::thinning(img_binary, img_thinning_ZS, ximgproc::THINNING_ZHANGSUEN);
+    ximgproc::thinning(img_binary, img_thinning_GH, ximgproc::THINNING_GUOHALL);
```

Comment:
Forget this last remark, it should indeed be there or at least include `using namespace cv::ximgproc` in the beginning of the file!